homeassistant/components/samsungtv/bridge.py

```diff
@@ -8,6 +8,7 @@
 from websocket import WebSocketException
 
 from homeassistant.const import (
+    CONF_DESCRIPTION,
```

Comment:
The description in this integration is a samsung library specific parameter. We don't want to use the Home Assistant common constants for that.

Reply:
I don't really get your point. What makes this parameter library specific compared to "host" or "name" where the HA constants are already used in that integration?

Reply:
We don't want to couple the Home Assistant api directly with the library api. I haven't looked at other constants.

Reply:
I get point in general, but again other `CONF_*` are used as well: https://github.com/home-assistant/core/blob/a8758ed3a143d2ce9d2b7fa94b5837b418746f7b/homeassistant/components/samsungtv/bridge.py#L129-L138

So either all of those parameters need to be completely integration-local or none should. IMHO "description" is not really `samsungtv` specific (otherwise I would have never proposed to move it to the HA global 😉) and should be treated as we do with "host", "port", etc.

Reply:
No. This PR is not about the samsung integration or about the other constants. We can fix that in another PR if needed.

The config attribute in the example above is passed to the Remote instance from the samsung library, so it's a library specific api.